apps/client/src/features/app-settings/panel/general-panel/ViewSettingsForm.tsx

```diff
@@ -14,11 +14,14 @@ import { preventEscape } from '../../../../common/utils/keyEvent';
 import { isOntimeCloud } from '../../../../externals';
 import * as Panel from '../../panel-utils/PanelUtils';
 
+const CodeEditorModal = lazy(() => import('./StyleEditorModal'));
```

Comment:
question: what do you think about lazy loading the `<CodeEditor>` instead?
I wonder if we can get it to load while we fetch the CSS?